src/ensenso.cpp

```diff
@@ -969,4 +969,40 @@ Result<Ensenso::CaptureParams> Ensenso::getCaptureParameters(bool crop_to_roi) {
 	return params;
 }
 
+bool Ensenso::isDumpingTree() {
+	return dump_tree_;
+}
+
+void Ensenso::enableNxLibLogging(std::string connection_folder_path, std::string debug_level, int log_file_size, bool dump_tree) {
```

Comment:
```suggestion
void Ensenso::enableDebugLogging(std::string connection_folder_path, std::string debug_level, int log_file_size, bool dump_tree) {
```

The fact that it uses nxlib is an implementation detail I would argue.

Reply:
Would also be nice to have a function to disable debug logging. That way we can start/stop debug logging when we want to.